usecase/getAppInformation.go

```diff
@@ -25,6 +21,7 @@ type App struct {
 }
 
 func GetAppsInformation(androidAppIds []string, iosAppIds []string) AppsInformation {
```

Comment:
Those are not `androidAppIds` annymore. But rather `appIdsOrDeveloperIds`... We might want to change this.